osCam/templates/user/add_config.html

```diff
@@ -0,0 +1,31 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Add Camera Configuration</title>
+        {% load static %}
+        {% include "core/bootstrap.html" %}
+        <link rel="stylesheet" href="{% static 'core/dash.css' %}">
+
+        {% load crispy_forms_field %}
+        {% load crispy_forms_tags %}
+    </head>
+        <body>
+            {% include "core/navigation.html" %}
+            <div class="Jumbotron"
+                style="color: rgba(15, 34, 75, 0.644); background: beige; margin-left: 30%; margin-right: 30%">
```

Comment:
style attributes should be defined in the django apps corresponding static/config/CSS file then reference the style for this <DIV> just as you linked `<link rel="stylesheet" href="{% static 'core/dash.css' %}">` If you think the style will only be applied this one time, then give the DIV an attribute of id="my_meaningful_id". If it will be used more than once give it a class="some_meaning_crispy_form_classname" as attribute. Then in your CSS you can do `div#my_meaningful_id{ color: rgba(15, 34, 75, 0.644); backgroud: beige; margin-left: 30%; margin-right: 30%; }` for the ID and for the class example just replace the div#my_meaningful_id with CSS class reference of  `.some_meaningful_crisply_form_classname { /*put the css style attributes here*/ }`. 

Again, use the div#some_id example if this style will only be used once and do the (dot) class (**.**some_class) if we can re-use the style accross multiple Forms.